alacritty_terminal/src/term/mod.rs

```diff
@@ -938,6 +938,11 @@ impl Term {
         self.dynamic_title = config.dynamic_title();
         self.auto_scroll = config.scrolling.auto_scroll;
         self.grid.update_history(config.scrolling.history() as usize, &self.cursor.template);
+
+        self.original_font_size = config.font.size;
+        if self.original_font_size == self.font_size {
```

Comment:
We need to check first and set it later, not set it first and then check. Otherwise we'll compare to the changed value.